pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-assemblies.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+
+[id="con_about-assemblies_{context}"]
+= About Assemblies
```

Comment:
@RichardHoch Andrew pointed out that we agreed on the convention "Overview of..." so this would be "Overview of Assemblies"